app/views/bulk_update/recommendations_uploads/_summary_bullets.html.erb

```diff
@@ -5,8 +5,14 @@
 <ul class="govuk-list govuk-list--bullet">
   <% if awardable_rows_count.positive? %>
     <li>
-      <%= pluralize(awardable_rows_count, "trainee") %> who will be
-      recommended for <%= qts_or_eyts(recommendations_upload.awardable_rows) %>
+      <%= case awardable_rows_count
+        when 1
+          "The trainee"
+        when 2
+          "Both trainees"
+        else
+          "#{awardable_rows_count} trainees"
+        end %> who will be recommended for <%= qts_or_eyts(recommendations_upload.awardable_rows) %>
```

Comment:
Ahh I think this is the wrong place. This content is fine as it was. The change needs to be made in `app/views/bulk_update/recommendations_uploads/_summary_all_dates.html.erb` i.e. when ALL trainees have been given dates and there are no errors.

Reply:
Then in the first case:
```
The trainee will be recommended for QTS.
```

In the second case:
```
Both trainees will be recommended for QTS.
```

For anything else (no change):
```
All x trainees will be recommended for QTS.
```